### PR TITLE
doc: matter: Updated data for the memory requirements

### PR DESCRIPTION
**Pull request by kkasperczyk-no**
Generated up to date information for the memory requirements for all the supported Matter samples and applications.

### CODE REVIEW

File: doc/nrf/protocols/matter/getting_started/hw_requirements.rst [hunk 1]
```diff
@@ -57,104 +57,107 @@ Values are provided in kilobytes (KB).
       The following table lists memory requirements for samples running on the :ref:`nRF52840 DK <programming_board_names>` (:ref:`nrf52840dk_nrf52840 <zephyr:nrf52840dk_nrf52840>`).
 
       +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | Sample                                                           |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
+      | Sample                                                           |   MCUBoot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```
Comment (wiba-nordic) [inline doc/nrf/protocols/matter/getting_started/hw_requirements.rst:60]:
```suggestion
      | Sample                                                           |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```
That shouldn't change.

File: doc/nrf/protocols/matter/getting_started/hw_requirements.rst [hunk 2]
```diff
@@ -57,104 +57,107 @@ Values are provided in kilobytes (KB).
...
+      | :ref:`Window Covering <matter_window_covering_sample>` (Release) |            28 |               682 |              4 |         32 |         746 |                             159 |
       +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
 
    .. tab:: nRF5340 DK
 
       The following table lists memory requirements for samples running on the :ref:`nRF5340 DK <programming_board_names>` (:ref:`nrf5340dk_nrf5340_cpuapp <zephyr:nrf5340dk_nrf5340>`).
 
       +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | Sample                                                           |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
+      | Sample                                                           |   MCUBoot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```
Comment (wiba-nordic) [inline doc/nrf/protocols/matter/getting_started/hw_requirements.rst:92]:
```suggestion
      | Sample                                                           |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```

File: doc/nrf/protocols/matter/getting_started/hw_requirements.rst [hunk 3]
```diff
@@ -57,104 +57,107 @@ Values are provided in kilobytes (KB).
...
       +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
 
-   .. tab:: Nordic Thingy:53
+   .. tab:: Thingy:53
 
       The following table lists memory requirements for samples running on the :ref:`Thingy:53 <programming_board_names>` (:ref:`thingy53_nrf5340 <zephyr:thingy53_nrf5340>`).
 
       +---------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | Sample                                                        |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
+      | Sample                                                        |   MCUBoot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```
Comment (wiba-nordic) [inline doc/nrf/protocols/matter/getting_started/hw_requirements.rst:124]:
```suggestion
      | Sample                                                        |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```

File: doc/nrf/protocols/matter/getting_started/hw_requirements.rst [hunk 4]
```diff
@@ -57,104 +57,107 @@ Values are provided in kilobytes (KB).
...
       +---------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
 
    .. tab:: nRF7002 DK
 
       The following table lists memory requirements for samples running on the :ref:`nRF7002 DK <programming_board_names>` (:ref:`nrf7002dk_nrf5340_cpuapp <nrf7002dk_nrf5340>`).
 
-
       +------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | Sample                                                     |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
+      | Sample                                                     |   MCUBoot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```
Comment (wiba-nordic) [inline doc/nrf/protocols/matter/getting_started/hw_requirements.rst:136]:
```suggestion
      | Sample                                                     |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by doublemis1 — APPROVED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Review by LuDuda — APPROVED** [review]
This is super critical for NCS 2,.5.0! Thank you LGTM :+1:

**Review by Damian-Nordic — APPROVED** [review]

**Pull request merged**